src/vaadin-login-overlay.html

```diff
@@ -79,15 +52,37 @@
 
         static get properties() {
           return {
+            /**
+             * Defines the application description
+             */
+            description: {
+              type: String,
+              value: 'Application description',
+              notify: true
```

Comment:
Is the description expected to be changed internally by the component? If not, no need for `notify: true` here